YMChat/String.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  String.swift
+//  YMChat
+//
+//  Created by Sankalp Gupta on 22/12/21.
+//
+
+import Foundation
+import UIKit
+extension String {
+    
+    var isImageType: Bool {
+        if let url = URL(string: self) {
+            do {
+                _ = try UIImage(data: Data(contentsOf: url))
+                return true
+            } catch {
+                
+            }
+        }
+        return false
```

Comment:
This is not required. We can only check if it is a valid URL. This image will be loaded in webcontainer not in native environment.

Also, this might fail for gif image. Not sure if UIImage supports gif out of the box

Reply:
Ok. I will make the required changes.